docs/pages/build-platforms/github-actions.md

```diff
@@ -40,6 +41,7 @@ jobs:
 
       - name: Create Release
         env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Wait, this isn't necessary per #1036: `GITHUB_TOKEN` is already in the `env`.

Reply:
In my testing I found that it wasn't there seem to be contradictory docs too